guides/common/assembly_deployment-path.adoc

```diff
@@ -0,0 +1,40 @@
+include::modules/con_deployment-path.adoc[]
+
+include::modules/con_installing-a-server.adoc[leveloffset=+1]
+
+include::modules/con_project-server-with-external-database.adoc[leveloffset=+2]
+
+include::modules/con_configuring-dns-dhcp-and-tftp.adoc[leveloffset=+2]
+
+include::modules/ref_overview-of-authentication-methods-in-project.adoc[leveloffset=+1]
+
+include::modules/con_planning-organization-and-location-context.adoc[leveloffset=+1]
+
+include::modules/con_installing-smart-proxies.adoc[leveloffset=+1]
```

Comment:
please adjust to "installing-smart-proxy-server". Smart Proxy = software component/Foreman Server and all Smart Proxy Servers; Smart Proxy Server = a decicated host that runs Smart Proxy.